freebridge-ai/main.py

```
@@ -0,0 +1,78 @@
+@app.on_event("startup")
+async def load_documents():
+    docs_dir = "./docs"
+    if not (os.path.exists(docs_dir) and os.listdir(docs_dir)):
+        print("ℹ️ docs 폴더가 비어있어 학습을 건너뜁니다.")
+        return
+
+    try:
+        # 중복 학습 방지: 데이터가 이미 있으면 중단
+        count = vectorstore._collection.count()
+        if count > 0:
+            print(f"ℹ️ 이미 {count}개의 벡터 데이터가 존재합니다. 자동 학습을 건너뜁니다.")
```

Ruff (0.15.2)

[warning] String contains ambiguous `ℹ` (INFORMATION SOURCE). Did you mean `i` (LATIN SMALL LETTER I)?

(RUF001)